docs/API.md

```diff
@@ -175,16 +175,11 @@ class App extends React.Component {
 ```
 
 ##### `components`
-Routes can define multiple components as an object of `name:component`
+Routes can define one or more named components as an object of `name:component`
 pairs to be rendered when the path matches the URL. They can be rendered
-by the parent route component with `this.props.children[name]`.
+by the parent route component with `this.props[name]`.
 
 ```js
-// think of it outside the context of the router, if you had pluggable
-// portions of your `render`, you might do it like this
-<App children={{main: <Users/>, sidebar: <UsersSidebar/>}}/>
```

Comment:
We'd rewrite this as e.g.

```js
<App main={<Users />} sidebar={<UsersSidebar />} />
```

It's just a good illustration of what this is making the router do.